src/components/atoms/UiTextarea/UiTextarea.vue

```diff
@@ -85,6 +99,18 @@ const resizeValue = computed(() => {
   }
   return props.resize ? 'both' : 'none';
 });
+const textareaElement = ref<HTMLTextAreaElement | null>(null);
+const textareaHeight = ref<number | undefined>(undefined);
+
+function dynamicHeight(element: HTMLTextAreaElement) {
```

Comment:
@sonya0504 maybe we can test this solution https://css-tricks.com/the-cleanest-trick-for-autogrowing-textareas/ WDYT?